types/post_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoStProofPartitions TODO(laser)
```

Comment:
PoStProofPartitions represents the number of partitions used when creating a PoSt proof, and impacts the size of the proof.